src/webhook_service/terraform/lambda.tf

```diff
@@ -159,6 +157,135 @@ module "get_webhook_docker_image" {
   }
 }
 
+####################################
+####################################
+####################################
+# GET /webhooks ####################
+####################################
+####################################
+####################################
+
+module "list_webhooks_lambda" {
+  source  = "terraform-aws-modules/lambda/aws"
+  version = "7.7.0"
+
+  function_name  = local.list_webhooks_function_name_and_ecr_repo_name
+  description    = "AWS Educate TPET ${var.service_hyphen} in ${var.environment}: GET /webhooks"
+  create_package = false
+  timeout        = 30
+
+  ##################
+  # Container Image
+  ##################
+  package_type = "Image"
+  # architectures = ["x86_64"] # or ["arm64"]
+  architectures = ["arm64"]
+  # image_uri     = "public.ecr.aws/lambda/python:3.8"
+  # image_uri     = module.list_emails_docker_image.image_uri
+  image_uri = module.list_webhooks_docker_image.image_uri
+
+  publish = true # Whether to publish creation/change as new Lambda Function Version.
+
+  environment_variables = {
+    "ENVIRONMENT"    = var.environment,
+    "SERVICE"        = var.service_underscore,
+    "DYNAMODB_TABLE" = var.dynamodb_table
+    "DYNAMODB_TABLE_TOTAL_COUNT" = var.dynamodb_table_total_count
+  }
+
+  allowed_triggers = {
+    AllowExecutionFromAPIGateway = {
+      service    = "apigateway"
+      source_arn = "${module.api_gateway.api_execution_arn}/*/*"
+    }
+  }
+
+  tags = {
+    "Terraform"   = "true",
+    "Environment" = var.environment,
+    "Service"     = var.service_underscore
+  }
+  ######################
+  # Additional policies
+  ######################
+
+  attach_policy_statements = true
+  policy_statements = {
+    dynamodb_crud = {
+      effect = "Allow",
+      actions = [
+        "dynamodb:BatchGetItem",
+        "dynamodb:BatchWriteItem",
+        "dynamodb:DeleteItem",
+        "dynamodb:GetItem",
+        "dynamodb:PutItem",
+        "dynamodb:Query",
+        "dynamodb:Scan",
+        "dynamodb:UpdateItem"
+      ],
+      resources = [
+        "arn:aws:dynamodb:${var.aws_region}:${data.aws_caller_identity.this.account_id}:table/${var.dynamodb_table}",
+        "arn:aws:dynamodb:${var.aws_region}:${data.aws_caller_identity.this.account_id}:table/${var.dynamodb_table}/index/SequenceNumberIndex",
```

Comment:
```suggestion
        "arn:aws:dynamodb:${var.aws_region}:${data.aws_caller_identity.this.account_id}:table/${var.dynamodb_table}/index/webhook-webhook_type-sequence_number-gsi",
```

Reply:
Changes of global secondary index have been applied
```
      resources = [
        "arn:aws:dynamodb:${var.aws_region}:${data.aws_caller_identity.this.account_id}:table/${var.dynamodb_table}",
        "arn:aws:dynamodb:${var.aws_region}:${data.aws_caller_identity.this.account_id}:table/${var.dynamodb_table}/index/webhook-webhook_type-sequence_number-gsi",
        "arn:aws:dynamodb:${var.aws_region}:${data.aws_caller_identity.this.account_id}:table/${var.dynamodb_table_total_count}",
      ]
```

Take a look at [lambda.tf](https://github.com/aws-educate-tw/aws-educate-tpet-backend/blob/SCRUM-157-list-webhooks/src/webhook_service/terraform/lambda.tf)